### PR TITLE
added filter page

### PR DESCRIPTION
**Pull request by srijani07**
Filters:
image

image

### CODE REVIEW

**Review by HarshnaNHaswani — APPROVED** [review]
Format your files using any code formatter,
Rest is good

File: filters.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://verve-ui.netlify.app/styles.css" />
+
```
Comment (Bibin-Antony) [inline filters.html:10]: you could use prettifier to enable you to fix these gap issues.

**Review by Bibin-Antony — APPROVED** [review]
seems fine to me. reduce unnecessary gaps in the code.

File: filters.html (hunk 1 above)
Comment (srijani07) [reply filters.html:12]: Done, thanks!

**Pull request merged**